tests/caasp/transactional_update.pm

```diff
@@ -69,55 +64,45 @@ sub check_reboot_changes {
 }
 
 sub run {
-    # FIXME: Make a different version of the update-test-security package for Tumbleweed, with a valid key, to make test #2 valid
-    my $ptfutsver     = '1.3';
-    my $ptfutsverlong = '5-1.3';
-    my $oriutsver     = '1.3';
-
-    # Define versions of update-test-security used in testing. CaaSP has different versions than Kubic.
-    if (check_var('DISTRI', 'caasp')) {
-        $ptfutsver     = '5.3.61';
-        $ptfutsverlong = '5-5.3.61';
-        $oriutsver     = '5.30.1';
-    }
-
     script_run "rebootmgrctl set-strategy off";
 
     get_utt_packages;
 
     record_info 'Install ptf', 'Install package - snapshot #1';
-    trup_call "ptf install update-test-trival/update-test-security-$ptfutsverlong.x86_64.rpm";
+    trup_call "ptf install" . rpmver('security');
     check_reboot_changes;
-    check_package "$ptfutsver";
+    check_package rpmver('in');
 
     record_info 'Update #1', 'Add repository and update - snapshot #2';
-    # FIXME: Disable test for now on Kubic, as impossible to change package versions ATM
-    if (check_var('DISTRI', 'kubic')) {
-        record_soft_failure('Test #2 Skipped on openSUSE Kubic');
-    }
-    else {
+    if (is_caasp 'caasp') {
         # Only CaaSP needs an additional repo for testing
-        if (check_var('DISTRI', 'caasp')) {
-            assert_script_run 'zypper ar utt.repo';
-        }
+        assert_script_run 'zypper ar utt.repo' if is_caasp 'caasp';
         trup_call 'cleanup up';
         check_reboot_changes;
-        check_package "$oriutsver";
+        check_package rpmver('up');
+    }
+    else {
+        # No update repositories ATM
+        record_soft_failure('Test skipped on openSUSE Kubic');
```

Comment:
Maybe someone else can confirmed, but for what I know `record_soft_failure` should reference a bugzilla number, no?
`record_info` is not enough in this case? Or if Kubic should do the test at the end, maybe a bug could be open.

Reply:
You are right, it's best practice - fixed